packages/ui/src/organisms/ProductCard/ProductCardTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, AriaAttributes } from 'react'
+
+export interface ProductCardTitleProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  'aria-label'?: AriaAttributes['aria-label']
+}
+
+const ProductCardTitle = forwardRef<HTMLDivElement, ProductCardTitleProps>(
```

Comment:
Maybe we can call it `ProductCardHeader` and use `<header> `, because we can pass title and description here. What do you think?

Reply:
@hellofanny great suggestion! It makes the component more general, implying that it may have things beyond the title (like description, for example)